recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1,5 @@
+SUMMARY = "adds gcrypt (libgcrypt) to systemd binary for forward secure sealing (ffs) journald"
+PACKAGECONFIG:append = " gcrypt"
+
+# Disable systemd-resolved service
+PACKAGECONFIG:remove = " resolved nss-resolve"
```

Comment:
What is this? This does not seem FSS related?

Reply:
Agreed - but it was in the file in meta-pelion-edge, added by Ari on a fairly generic check-in comment.
I can take it out until we understand what problem it fixes...

Reply:
I've removed this part as it is not related to logging.

Reply:
https://layers.openembedded.org/layerindex/branch/master/recipes/?q=nss-resolved does not find anything either, that's strange. Once again testament why you should really use some time with the git commits, they can save a LOT of time later on if done right.